test/setup-env.ts

```diff
@@ -0,0 +1,7 @@
+process.env['INPUT_AM_URL'] = 'some-am-url';
+process.env['INPUT_USERNAME'] = 'admin';
+process.env['INPUT_PASSWORD'] = 'password';
```

Comment:
I'm assuming we won't actually store any secrets in code like this, yeah?

Reply:
Nah, this is purely for the github actions/core mocking.